static/js/components/CandidateList.jsx

```diff
@@ -41,13 +46,55 @@ const useStyles = makeStyles(() => ({
   spinnerDiv: {
     paddingTop: "2rem",
   },
+  infoItem: {
+    display: "flex",
+    "& > div": {
+      paddingLeft: "0.25rem",
+    },
+    flexFlow: "row wrap",
+  },
+  saveCandidateButton: {
+    margin: "0.5rem 0",
+  },
+  thumbnails: (props) => ({
+    minWidth: props.thumbnailsMinWidth,
+  }),
+  info: {
+    minWidth: 0,
+  },
 }));
 
+// Hide built-in pagination and tweak responsive column widths
+const getMuiTheme = (theme) =>
+  createMuiTheme({
+    overrides: {
+      MUIDataTableFooter: {
+        root: {
+          display: "none",
+        },
+      },
+      MUIDataTableBodyCell: {
+        stackedHeader: {
+          verticalAlign: "top",
+        },
+        stackedCommon: {
+          [theme.breakpoints.down("sm")]: { width: "calc(75%)" },
```

Comment:
@stefanv's style guide suggests to only use up, see [here](https://skyportal.io/docs/styling.html?highlight=breakpoint#responsive-design)

Reply:
@dmitryduev This overrides specifically defined styling in the MUIDataTable package itself, so I'm not sure if there's a straightforward to not use the down here. I think even if I wrote my own rules for the tables using up, I would need to clear the rules defined with down, hence using down in our code anyways.

Looking at your PR for the other changes now

Reply:
Actually, I guess that doesn't matter since you removed the headers completely on the small screens and that styling was to reduce the header sizes anyways.

Reply:
👍